setup.py

```diff
@@ -235,13 +234,10 @@ def create_dir_symlink(src, dest):
         nccl_version = ".".join(str(torch.cuda.nccl.version())[:2])
     else:
         nccl_version = ".".join(map(str, torch.cuda.nccl.version()[:2]))
-    if hasattr(torch.cuda, 'is_bf16_supported'):
-        bf16_support = torch.cuda.is_bf16_supported()
```

Comment:
The `bf16_support` value is only used in unit testing - perhaps we could add a try...except block around this?

Reply:
My first commit would be sufficient to prevent the crash because it checks whether CUDA is available.

Reply:
The bf16 support is only utilized in unit tests - so it shouldn't affect users that build/run on different hardware

Reply:
I know it's set up that way in CI, but if a developer runs the unit tests on a different machine than the build, it could still cause problems.